tests/test_desktop_control.py

```
@@ -0,0 +1,145 @@
+    reached = await app.state.desktop_command_broker.emit(
+        "system", DesktopCommand(kind="open-app", payload={"app": "projects"})
+    )
+    assert reached == 1, "the SSE subscriber should be the sole 'system' desktop"
+    try:
+        await asyncio.wait_for(done.wait(), timeout=3.0)
+    finally:
+        task.cancel()
+        try:
+            await task
+        except (asyncio.CancelledError, Exception):
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)